src/components/LayoutEmbed.js

```diff
@@ -177,24 +179,25 @@ export default function Layout({
           <SearchFilter />
         </Box>
       )}
-      <Box
-        onClick={handleFullScreen}
-        sx={{
-          position: 'absolute',
-          zIndex: '9999',
-          top: '0px',
-          right: '0px',
-          margin: '20px',
-          cursor: 'pointer',
-        }}
-      >
-        <SvgIcon
-          component={MinIcon}
-          inheritViewBox
-          sx={{ height: 52, width: 52 }}
-        />
-      </Box>
-
+      {isEmbed || (
```

Comment:
Should this be an OR? Now it will always render the elements

Reply:
since logical operators short circuit, it should only return the component if isEmbed is falsey